banzai/tests/test_image.py

```diff
@@ -1,7 +1,51 @@
 from banzai.images import Image
-from banzai.tests.utils import FakeContext
+from banzai.tests.utils import FakeContext, FakeImage
+import numpy as np
 
+rs = np.random.RandomState(0)
```

Comment:
This has no real affect in pytest. This should be a fixture like in test_dark_normalizer.

Reply:
Fixed